core/templates/components/summary-tile/learner-story-summary-tile.component.html

```diff
@@ -0,0 +1,110 @@
+<mat-card class="oppia-story-dashboard-card">
+  <a [href]="getStoryLink()"
```

Comment:
Whats the time-complexity of this getStoryLink function. If it's greater than O(1), can you come up with a way to precompute this (and recompute using ngOnChanges).

Reply:
The time complexity of this is O(1).